### PR TITLE
[test](load) injection cases should check Exception is thrown

### PR DESCRIPTION
**Pull request by kaijchen**
### What problem does this PR solve?

Problem Summary:

Some injection cases did not check exception is thrown.
This may cause false positives. 

### Check List (For Author)

- Test 
    - [x] Regression test
    - [ ] Unit Test
    - [ ] Manual test (add detailed scripts or steps below)
    - [ ] No need to test or manual test. Explain why:
        - [ ] This is a refactor/code format and no logic has been changed.
        - [ ] Previous test can cover this change.
        - [ ] No code files have been changed.
        - [ ] Other reason 

- Behavior changed:
    - [x] No.
    - [ ] Yes. 

- Does this need documentation?
    - [ ] No.
    - [ ] Yes. 

### Check List (For Reviewer who merge this PR)

- [ ] Confirm the release note
- [ ] Confirm test cases
- [ ] Confirm document
- [ ] Add branch pick label

### CODE REVIEW

**Comment by kaijchen** [comment]
run buildall

**Comment by kaijchen** [comment]
run buildall

File: regression-test/suites/fault_injection_p0/test_load_stream_back_pressure_fault_injection.groovy [hunk 1]
```diff
@@ -71,67 +71,34 @@ suite("test_load_stream_back_pressure_fault_injection", "nonConcurrent") {
...
-            }
+        kill_thread.start()
+        try {
+            def res = sql "insert into test select * from baseall where k1 <= 3"
+            logger.info(res.toString())
+            assertTrue(false, "Expected exception to be thrown")
+        } catch(Exception e) {
+            logger.info(e.getMessage())
+            assertTrue(e.getMessage().contains("wait flush token back pressure time is more than load_stream_max_wait_flush_token_time"))
+            //assertTrue(e.getMessage().contains("Communications link failure"))
```
[the remark this answers is not in the document]
Comment (kaijchen) [reply regression-test/suites/fault_injection_p0/test_load_stream_back_pressure_fault_injection.groovy:95]: the previous error message seems incorrect

**Comment by kaijchen** [comment]
run buildall

**Review by liaoxin01 — APPROVED** [review]
LGTM

**Review by dataroaring — APPROVED** [review]
LGTM

**Pull request merged**